### PR TITLE
Added List System Containers

### PR DESCRIPTION
**Pull request by tasherif-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/tests/settings/settings_fake.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 CLIENT_SECRET = "00000000-0000-0000-0000-000000000000"
 
 ACCOUNT_URL_SUFFIX = 'core.windows.net'
-RUN_IN_LIVE = "False"
-SKIP_LIVE_RECORDING = "True"
+RUN_IN_LIVE = "True"
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-blob/tests/settings/settings_fake.py:25]: Do you need to revert these 2 lines?

**Review by seanmcc-msft — COMMENTED** [review]
What about tests for Data Lake?

File: sdk/storage/azure-storage-blob/tests/settings/settings_fake.py (hunk 1 above)
Comment (tasherif-msft) [reply sdk/storage/azure-storage-blob/tests/settings/settings_fake.py:25]: yep yep

**Comment by tasherif-msft** [comment]
TODO: add datalake tests

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**